qiskit_superstaq/superstaq_backend.py

```diff
@@ -11,13 +11,57 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
+import re
 from typing import Any, Dict, List, Optional, Union
 
 import qiskit
 
 import qiskit_superstaq as qss
 
 
+def validate_backend(backend_str: str) -> None:
+    backend_prefixes = [
```

Comment:
throughout pr: we should use "target" whenever referring to a superstaq-formatted device string
```suggestion
def validate_target(target: str) -> None:
    target_prefixes = [
```
(or maybe `vendor_prefixes` for the second line?)